server/utils/utils.ts

```diff
@@ -159,3 +159,9 @@ export const formatAppointmentNote = (inputNote?: string): string => {
 export const orElse = (input: string, alternative: string): string => {
   return input || alternative
 }
+
+export const isBeforeDate = (input: string): boolean => {
```

Comment:
Suggestion: Maybe something like `isDateInPast` would describe what this does better?